test/map.py

```diff
@@ -0,0 +1,88 @@
+import math 
+import numpy as np
+import pandas as pd
+import utm
+
+class MAP():
+  """
+  MAP loads data from a csv file (region.csv) placed in the data folder
+
+  """
+  _data = {}
+  _fileName = './data/region.csv'
+  
+  def __init__(self):
+    # loading data
+    self._data = pd.read_csv( self._fileName )
+    print( 'Data successfully loaded!' )
+
+  def LatLon(self):
+    "Returns raw data"
+    return( self._data )
+
+  def Utm(self):
+    "Converts latitude and longitude geolocation to utm"
+    # It has to be reviewed: The data is in a different format (+ddddddd.ddddd). Check utm.from_latlon documentation
+    
+    self._data[ ['x0', 'y0', 'utmZoneNumber', 'utmZoneLetter'] ] = self._data.apply( 
+      lambda row: pd.Series( 
+        list( utm.from_latlon( row['lat'], row['lon'] ) )
+      ),
+      axis = 1 
+    )
+    return (
+      self._data
+    )
+
+  def MdtGradient(self, data, step):

```

Comment:
Duplicated code. Perhaps the MDT deserves a module for itself.

Reply:
Thank you for the review! 
All the files inside the test folder is duplicated. I had a problem with my laptop, so I'm using another machine which I can't install git. I tried to delete that folder, however I couldn't find a way to do it here in the Github page. Is it possible?
Anyway, I might get my new laptop this week, then I'll fix this problem. 

Thanks again!